src/Core/src/Platform/Windows/ViewExtensions.cs

```diff
@@ -135,8 +138,13 @@ public static void UpdateFlowDirection(this FrameworkElement platformView, IView
 			}
 		}
 
-		public static void UpdateAutomationId(this FrameworkElement platformView, IView view) =>
-			AutomationProperties.SetAutomationId(platformView, view.AutomationId);
+		public static void UpdateAutomationId(this FrameworkElement platformView, IView view)
+		{
+			if (!view.IsPlatformViewNew || view.AutomationId is not null)
```

Comment:
Is the default for `AutomationId` `null` or `string.Empty`? I don't know the answer, so just checking.